.scripts/generate-config-doc.js

```diff
@@ -1,149 +1,353 @@
 const fs = require('fs')
+const { type } = require('os')
 const path = require('path')
 const schemaPath = path.join(__dirname, '../../tauri/tooling/cli/schema.json')
-const schema = JSON.parse(fs.readFileSync(schemaPath).toString())
+const schemaString = fs
+  .readFileSync(schemaPath)
+  .toString()
+  .replaceAll('<', '&lt;') // Fixes opening HTML brackets
+  .replaceAll('>', '&gt;') // Fixes closing HTML brackets
```

Comment:
These ones actually break most of the usages since they are in "backtick blocks". Tricky to to handle both cases here 🤔 
Not counting links, only 2 occurrences without backticks.

Reply:
yeah that's the 2 occurrences i was talking about :P